### PR TITLE
fix #47941 projekt-gesund-leben.de 

### LINKED ISSUE #47941: projekt-gesund-leben.de
### Issue URL (Annoyance)
[https://www.projekt-gesund-leben.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.projekt-gesund-leben.de%2F)
### Comment
Username: @Morku
> Cookie Pop-up
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Referrer:
> 
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/popup-blocker/2.5/popupblocker.user.js
> https://userscripts.adtidy.org/release/assistant/4.3/assistant.user.js
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (180)
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard German
Custom filter subscriptions: |

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#47941

### CODE REVIEW

**Comment by Alex-302** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/49544/files#diff-8c0b559f58de5d465b7a5517ad6845d2R1297-R1299
I don't see these elements

**Comment by Sergey-Lyapin** [comment]
> https://github.com/AdguardTeam/AdguardFilters/pull/49544/files#diff-8c0b559f58de5d465b7a5517ad6845d2R1297-R1299
> I don't see these elements

Check these pages for the first rule
https://www.projekt-gesund-leben.de/2018/02/zuckerfreie-einkaufsliste/
https://www.projekt-gesund-leben.de/ueber-das-projekt/

`:not([itemprop="mainContentOfPage"])` is because of these pages
https://www.projekt-gesund-leben.de/saisonkalender/
https://www.projekt-gesund-leben.de/zuckerfrei/

for the rest two rules
https://www.projekt-gesund-leben.de/retreats/

**Comment by Sergey-Lyapin** [comment]
@BlazDT please take a look

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1294,6 +1294,9 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
 !
 ngadget.ru###revsubscribe
 digitalcameraworld.com###newsletterWidget
+projekt-gesund-leben.de#?#.container > :not([itemprop="mainContentOfPage"]) > .post-entry > div.entry-content-wrapper > div.flex_column > section.avia_codeblock_section:has(> div.avia_codeblock > div[class^="_form"] > form._inline-form > div._form-content > div._form_element > div._html-code:contains(Trag dich in meine E-Mail-Liste))
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/annoyances.txt]: Change ext-CSS rule to 
`projekt-gesund-leben.de##._form_41`

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -67,6 +67,9 @@ cherry.de,ba-computer.at,stoertebeker.com,inateck.com,biershop-bayern.de,nkd.com
 tweaktown.com##.cookie-prompt
 fandom.com##.wds-banner-notification__container
 660citynews.com##.rdm-daac-notice-template
+projekt-gesund-leben.de##.pixelmate-big-wrapper
+projekt-gesund-leben.de##.pixelmate-revoke
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/cookies_specific.txt]: is part of previous rule.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 3]
```diff
@@ -67,6 +67,9 @@ cherry.de,ba-computer.at,stoertebeker.com,inateck.com,biershop-bayern.de,nkd.com
...
+elopage.com##.consent-management-simple
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/cookies_specific.txt]: Not shown.

File: AnnoyancesFilter/sections/annoyances.txt [hunk 4]
```diff
@@ -1294,6 +1294,9 @@ bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.
...
+projekt-gesund-leben.de#?#.post-entry > div.entry-content-wrapper > div.av-special-heading:has(> h2:contains(Trag dich in unseren Verteiler ein))
+projekt-gesund-leben.de#?#.avia_codeblock > div[class^="_form"] > div > form._form:has(div._form-content > div._button-wrapper > button._submit:contains(JETZT ANMELDEN))
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/annoyances.txt]: projekt-gesund-leben.de##._form_17

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @BlazDT 
<details><summary> Here it is</summary></details>

Comment (BlazDT) [reply AnnoyancesFilter/sections/cookies_specific.txt]: That rule is OK, but I suggested exchange of ext-CSS rules for normal CSS rule.

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/cookies_specific.txt]: There is only one rule with ext-CSS now

**Review by BlazDT — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**